extension/webviews/components/ViewerTopBar.svelte

```diff
@@ -10,18 +11,43 @@
   const switchToInvitePage = () => {
     $page = Page.InviteUser;
   };
+
+  const showInviteButton = $page === Page.ThreadsViewer || $page === Page.ReplyViewer;
+
+  const dropdownOptions = [
+    {
+      key: 'logout',
+      label: 'Logout',
+      handler: logout,
+    },
+    ...(showInviteButton
+      ? [
+          {
+            key: 'invite',
+            label: 'Invite',
+            handler: switchToInvitePage,
+          },
+        ]
+      : []),
+  ];
+
+  const handleSearchButtonClick = () => {
+    $page = Page.SearchViewer;
+  };
 </script>
 
 <div class="header">
-  <div>Welcome <span class="name-header">{username}</span></div>
+  {#if $page !== Page.SearchViewer}
```

Comment:
I don't like this. This condition should be handled in Sidebar.svelte I suppose?